test/apps/iodemo/ucx_wrapper.h

```diff
@@ -56,15 +56,16 @@ class UcxLog {
     ~UcxLog();
 
     template<typename T>
-    const UcxLog& operator<<(const T &t) const {
+    UcxLog& operator<<(const T &t) {
         if (_enable) {
-            std::cout << t;
+            (*_ss) << t;
         }
         return *this;
     }
 
 private:
     const bool               _enable;
+    std::stringstream*       _ss;
```

Comment:
*_ss

Reply:
please see:
https://github.com/yosefe/ucx/blob/integration3/test/apps/iodemo/ucx_wrapper.h#L169
and
https://github.com/yosefe/ucx/blob/integration3/test/apps/iodemo/ucx_wrapper.h#L248
I used the same convension

Reply:
then need to fix that places as well :)